docs/sources/clients/promtail/troubleshooting.md

```diff
@@ -43,6 +44,9 @@ If no changes are applied during a stage, that is usually an indication of a mis
 The `--inspect` flag should not be used in production, as the calculation of changes between pipeline stages negatively
 impacts Promtail's performance.
 
+Please note that only **the first configuration entry** under the `scrape_config` item will be applied while using the `--stdin` mode.
```

Comment:
I like this addition. I think maybe instead of changing the sections above, you could add another section here, like "to test a file multiple scrape_configs" or something, but the previous examples are valid without a config file.